src/app/news/news-list-item.component.html

```
@@ -88,8 +88,8 @@
     }
   </mat-card-content>
   <mat-card-actions class="flex-vertical-center">
-    <img [src]="item.avatar" class="cursor-pointer profile-outline" (click)="openMemberDialog(item.doc.user)">
-    <button mat-button type="button" class="cursor-pointer user-name-truncate margin-lr-8" (click)="openMemberDialog(item.doc.user)">
+    <img [src]="item.avatar" tabindex="0" role="button" class="cursor-pointer profile-outline" (keydown.enter)="openMemberDialog(item.doc.user, $event)" (keydown.space)="openMemberDialog(item.doc.user, $event)" (click)="openMemberDialog(item.doc.user, $event)">
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)